serverless/sql-databases/concepts.mdx

```diff
@@ -20,7 +20,7 @@ Autoscaling can automatically scale your database resources up or down, dependin
 
 ## Backup
 
-A database backup is a copy of the data stored in the database (including data structure, such as tables). Database backups in Serverless SQL Database are logical backups (as opposed to raw disk volume backups, also called "snapshots"). Database backups can be restored in the current database or any other PostgreSQL database and allows to recover from database crash, data corruption, or data loss.
+A database backup is a copy of the data stored in the database (including data structure, such as tables). Database backups in Serverless SQL Database are logical backups (as opposed to raw disk volume backups, also called "snapshots"). Database backups can be restored in the current database or any other PostgreSQL database and, allow recovery from database crashes, data corruption, or data loss.
```

Comment:
Just wondering about the wording of "Serverless SQL Database" (if the "d" should be capitalized or not)